<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessClass</class>
 <widget class="QMainWindow" name="ChessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionCreate_the_connection"/>
    <addaction name="actionConnect_to_server"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionAdmit_defeat"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_the_connection">
   <property name="text">
    <string>Create the connection</string>
   </property>
  </action>
  <action name="actionConnect_to_server">
   <property name="text">
    <string>Connect to server</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionAdmit_defeat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Admit defeat</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="chess.qrc"/>
 </resources>
 <connections/>
</ui>
